src/Operation/Count.php

```diff
@@ -34,7 +34,7 @@
  * @see \MongoDB\Collection::count()
  * @see http://docs.mongodb.org/manual/reference/command/count/
  */
-class Count implements Executable
+class Count implements Explainable
```

Comment:
Extending multiple classes is not permitted, but we can implement multiple interfaces. I realize Executable is `@internal` and not used for anything, but we don't need to remove it in this PR.